receptor_satellite/worker.py

```diff
@@ -18,16 +60,76 @@ def __aiter__(self):
             raise StopAsyncIteration
         return await self.get()
 
+    async def ack(self, playbook_run_id):
+        payload = {
+            'type': 'playbook_run_ack',
+            'playbook_run_id': playbook_run_id
+        }
+        await self.put(payload)
+
+    async def playbook_run_update(self, host, output):
+        payload = {
+            'type': 'playbook_run_update',
+            'playbook_run_id': host.run.playbook_run_id,
+            'sequence': host.sequence,
+            'host': host.name,
+            'console': output
+        }
+        await self.put(payload)
+
+    async def playbook_run_finished(self, host):
+        payload = {
+            'type': 'playbook_run_finished',
+            'playbook_run_id': host.run.playbook_run_id,
+            'host': host.name,
+            'status': 'success' # TODO
+        }
+        await self.put(payload)
+
+
+class Config:
+    def __init__(self, text_updates = False, text_update_interval = 5000, text_update_full = True):
+        self.text_updates = text_updates
+        self.text_update_interval = text_update_interval
+        self.text_update_full = text_update_full
+
+
+    def from_raw(raw = {}):
+        return Config(raw['text_updates'], raw['text_update_interval'], raw['text_update_full'])
+
+
+class Host:
```

Comment:
Often when you have a class with two methods, one of which is __init__, you probably don't need a class.